cases-SEP31/util/sep10.js

```diff
@@ -0,0 +1,26 @@
+import StellarSDK from "stellar-sdk";
```

Comment:
Is this just a copy? Why not use the original?

Reply:
it was more different before, updated to use original